doc/man3/OpenSSL_version.pod

```diff
@@ -52,22 +52,22 @@ Deprecated:
 =head2 Macros
 
 The three macros B<OPENSSL_VERSION_MAJOR>, B<OPENSSL_VERSION_MINOR> and
-B<OPENSSL_VERSION_PATCH> represent the three parts of a 3 numbered version
-number, MAJOR.MINOR.PATCH.
+B<OPENSSL_VERSION_PATCH> represent the three parts of a version
+number, B<MAJOR.MINOR.PATCH>.
 
 The macro B<OPENSSL_VERSION_PRE_RELEASE> is an added bit of text that,
 when defined, indicates that this is a pre-release version, such as
-C<"-dev"> for an ongoing development snapshot, C<"-alpha3"> for an
-alpha release, etc...
+C<"-dev"> for an ongoing development snapshot or C<"-alpha3"> for an
+alpha release.
 The value must be a string.
 
-The macro B<OPENSSL_VERSION_BUILD_METADATA> is extra metadata, reserved
-for other parties (examples: C<"+fips">, C<"+vendor.1">).
+The macro B<OPENSSL_VERSION_BUILD_METADATA> is extra information, reserved
+for other parties, such as C<"+fips">, or C<"+vendor.1">.
 The OpenSSL project will not touch this macro.
 The value must be a string.
 
 B<OPENSSL_VERSION_STR> is a convenience macro to get the short version
-number string, "MAJOR.MINOR.PATCH".
+number string, C<MAJOR.MINOR.PATCH>.
```

Comment:
I'm ok with the markup, but the quotes should stay because it really is a string.

Reply:
@levitte and @t8m please come to some agreement and tell me what to do.

My preference is that since it is output, notably *literal* output, it should be in ```C<>``` markup.  But the two of you decide.

Reply:
What part of "I'm ok with the markup" got lost here?  What I'm proposing is `C<"MAJOR.MINOR.PATCH">`...  although `C<"I<MAJOR>.I<MINOR>.I<PATCH>">` would probably be more accurate.

Why do I propose this change?  Because `OPENSSL_VERSION_STR` *is* literally defined to a string, i.e.:

``` C
#define OPENSSL_VERSION_STR "3.0.0"
```

not:

``` C
#define OPENSSL_VERSION_STR 3.0.0
```

Do you see the difference?

Reply:
That helps, but still not explicit enough for me, sorry.  Do you want *all* of them changed, or just that one?  For example, at line 104:
```
The compiler flags set for the compilation process in the form
"compiler: ..."  if available or "compiler: information not available"
 otherwise.
```

Add ```C<>``` around the quotes or, as I currently have it, use ```C<>``` instead of the quotes?

Reply:
Line 104 isn't about a macro definition, it's the returned value from `OpenSSL_version`, which *is* a string, but there are no literal quote in that string, and I could frankly go either way with those, depending on if you want express that it's a string (quotes included) or just display the contents of that string (quotes excluded).

Reply:
I changed the one instance at line 70.  I also changed a couple "version number" to "version identifier"  Fixup commit pushed.